src/Intercom/Clients/UsersClient.cs

```diff
@@ -335,20 +335,25 @@ public User IncrementUserSession(String id)
             return result.Result;
         }
 
-        public User IncrementUserSession(User user)
+        public User IncrementUserSession(String id, List<String> companyIds)
```

Comment:
This allows us to pass in a list of company_ids to increase the session as the same time as the user's. Fixed #105.